crates/compiler/load_internal/src/file.rs

```diff
@@ -3952,17 +3952,25 @@ fn parse_header<'a>(
             },
             parse_state,
         )) => {
+            let module_name = match opt_expected_module_name {
+                Some(pq_name) => arena.alloc_str(pq_name.as_inner().as_str()),
+                None => {
+                    // [modules-revamp] [privacy-changes] TODO: Support test/check on nested modules
```

Comment:
Do we currently support nested modules?

Reply:
I can remove this comment, but it was simply copied from a different part of the file.

Reply:
I'm not sure what we mean by nested modules, if it means within the same file then definitely not, if it means in a subfolder in the package's folder then I think we do support this.